### PR TITLE
adding MP3 Pixel tracker

### PR DESCRIPTION
**Pull request by Jason-Bee**
adding a whole new layout which is more organized plus a lot of newly made pixel icons, thanks to reggie for letting me re-use some older ones and abyss for the huge help!

### CODE REVIEW

**Comment by Jason-Bee** [comment]
Unbenannt
here a picture of the finished product

**Comment by Miepee** [comment]
I'll leave it up to the Corruption Maintainers to decide, but a few comments on my side (which I wouldn't be blocking the PR for)
- I don't mean to downplay your work here, but some of the sprites really give a feeling of "took sprites and threw them together", due to how inconsistent they look. Like Morph Ball having different shading than hyper ball, which all look completely different to the spider ball, command visor sticking out like a sore thumb, or scan visor having no shading at all compared to xray.
- The grapple upgrades look really off to me right now. Currently they give me more a feeling of "trident" than "grapple", which I assume is due to missing shading, the neck being a bit too long, and the base being invisible in non-collected mode

**Comment by Jason-Bee** [comment]
" Like Morph Ball having different shading than hyper ball, which all look completely different to the spider ball,"
 due to colour difference, fixed the spider issue tho to keep some consistency.
 
 command visor sticking out like a sore thumb, or scan visor having no shading at all compared to xray.
 Scan shading is adjusted, command and Xray cant be changed. xray already has a lot of shading but white on more white is hard to read, changing command would take away the readability of command compared to Combat.
 
 The grapple upgrades look really off to me right now. Currently they give me more a feeling of "trident" than "grapple"
yea i noticed that aswell after you pointed it out, updated the handle to be more fitting, not perfect. but for sure better then before @Miepee

**Comment by Jason-Bee** [comment]
image

**Comment by ThanatosGit** [comment]
Uh, anyone feels responsible for this?

**Comment by henriquegemignani** [comment]
I kinda don't want more themes for any game before having a decent theme selector, but corruption has just one theme so far

**Comment by Miepee** [comment]
Abyss was working on redrawing some sprites, so there will probably be more commits pushed to this

**Comment by Miepee** [comment]
I'll mark this as draft until then.

File: randovania/data/gui_assets/tracker/pixel-icons/mp3/Command_visor.png
Comment (Miepee) [inline randovania/data/gui_assets/tracker/pixel-icons/mp3/Command_visor.png:1]: Please have a consistent naming scheme for your files.
I don't care if you decide on lower_case_snake, Upper_Case_Snake, PascalCase or camelCase, as long as your files are consistent with it.

**Review by Miepee — CHANGES_REQUESTED** [review]

File: randovania/data/gui_assets/tracker/pixel-icons/mp3/Command_visor.png
Comment (Miepee) [reply randovania/data/gui_assets/tracker/pixel-icons/mp3/Command_visor.png:1]: This has not been fixed. The files still have an inconsistent naming scheme.

**Comment by Miepee** [comment]
@Jason-Bee Bump

File: .vscode/settings.json
Comment (Miepee) [inline .vscode/settings.json:1]: please remove this file

**Review by Miepee — CHANGES_REQUESTED** [review]

File: randovania/data/gui_assets/tracker/prime3-Pixel.json
Comment (Miepee) [inline randovania/data/gui_assets/tracker/prime3-Pixel.json:1]: please rename this file to `prime3-pixel.json`  or it will fail loading on linux and macos.

**Review by Miepee — CHANGES_REQUESTED** [review]

**Review by Miepee — APPROVED** [review]
no oneelse feels responsible, and looks fine to me, so i'll merge

**Pull request merged**
